packages/frontend-2/components/dashboard/Webflow/Wrapper.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <section v-if="!error">
+    <h2 class="text-heading-sm text-foreground-2 mb-4">Blog</h2>
+    <div class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 gap-4">
+      <DashboardWebflowCard
+        v-for="webflowItem in webflowItems"
+        :key="webflowItem.id"
+        :webflow-item="webflowItem"
+      />
+    </div>
+  </section>
+  <section v-else />
```

Comment:
We should probably have an error/empty state in here, in case we couldn't fetch the posts

Reply:
I did consider this, but thought showing the "Blog" title and then saying "Could not fetch" or "No posts available" was a worse UX.

I don't have a strong opinion on this though. What do you think @benjaminvo ?